bin/mark_changepoints_in_json

```diff
@@ -65,13 +65,15 @@ class Segments(object):
     """A list of Segments for a whole run sequence.
     """
 
-    def __init__(self, delta, length, steady_state, cpts, means, variances, data, outliers):
+    def __init__(self, delta, half_bound, steady_state, length, cpts, means,
+                 variances, data, outliers):
         self.delta = delta
+        self.half_bound = half_bound
```

Comment:
Later we subtract it from `1.0` to make the lower bound, and add it to `1.0` to make the upper bound, so it is a half-bound.